docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+error can exceed the average error by small amounts.
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+**Note that the metric names below are subject to change without notice.**
+
+### P_00 readout error
+*   Metric key: single_qubit_p00_error
+*   Metric key: parallel_p00_error
+
+The p_00 is defined as the probability that the state is measured as |0⟩ after
+being prepared in the |0⟩ state.  The p_00 error is defined as one minus this
+result.
+
+There are several sources of error in this model.  This error is primarily
+composed of error in measurement (readout) of the qubit while in the ground
+state.  However, this probability also contains the error than the qubit was not
+reset into the |0⟩ ground state properly.  This is often called the SPAM (state
+preparation and measurement) error.
+
+The single qubit error is when the readout is measured in isolation (only one
+qubit is measured at a time), while the parallel error is taken for all qubits
+at the same time.
+
+### P_11 readout error
+*   Metric key: single_qubit_p11_error
+*   Metric key: parallel_p11_error
+
+The p_11 is defined as the probability that the state is measured as |1⟩ after
+being prepared in the |1⟩ state.  The p_11 error is defined as one minus this
+result.
+
+This is dominated by the error in measurement (readout) of the qubit, but it
+implicitly contains several different types of error.  Also possible is that the
+excited state |1⟩ was not prepared correctly or that the state decayed before
+measurement.  This error is generally expected to be higher than the P_00 error.
+
+The single qubit error is when the readout is measured in isolation (only one
+qubit is measured at a time), while the parallel error is taken for all qubits
+at the same time.
+
+### Readout separation error
+*   Metric key: single_qubit_readout_separation_error
+
+When measured by the system, the |0⟩ and |1⟩ states manifest as outgoing analog
+signals.  These signals must be interpreted as signifying one state or the
+other.  Since these analog signals are continuous distributions, there will be
+some statistical overlap in the two distributions that would be theoretically
+impossible to distinguish.  This is classified as the separation error, and is
+calculated by fitting Gaussian distributions to the signals prepared in the
+|0⟩ state and |1⟩ state and calculating the overlap between the two
+distributions.
+
+### Isolated 1 qubit randomized benchmark error: 
+*   Metric key: single_qubit_rb_average_error_per_gate
+*   Metric key: single_qubit_rb_pauli_error_per_gate
+*   Metric key: single_qubit_rb_incoherent_error_per_gate
+
+Single qubit gate error is estimated using randomized benchmarking by taking
+sequences of varying length of the 24 gates within the Clifford group
+(those that preserve the Pauli group under conjugation) then applying then
```

Comment:
nit: repeated words: "then applying then applying the inverse", should be "then applying the inverse"

Reply:
Done.